test/manipulation.test.js

```diff
@@ -743,7 +743,15 @@ describe('manipulation', function() {
       Todo = db.define('Todo', {
         content: String,
       });
-      db.automigrate(['Post', 'Todo'], done);
+      Person = db.define('Person', {
+        name: String,
+        gender: String,
+        married: Boolean,
+        age: {type: Number, index: true},
+        dob: Date,
+        createdAt: {type: Date, default: Date},
```

Comment:
Could not agree more, filed an issue for it: https://github.com/strongloop/loopback-datasource-juggler/issues/1317